### ISSUE TITLE
goctl 8月28日 新增的API生成代码的自定义类型有误

### ISSUE THREAD

**Issue by qyvip** [zeromicro/go-zero#2359]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior, if applicable:

1. The code is
以looklook中定义的api配置生成为例，实际应该生成如下代码，但目前生成的是2中的错误代码
应该是tools/goctl/api/gogen/util.go中的getRefProperty方法代码有问题

   ```go
   type CommentListResp struct {
	List []HomestayComment `json:"list"`
   }
   ```

2. The error is

   ```
   type CommentListResp struct {
	List struct {
	}
    }
   ```

**Expected behavior**

**Screenshots**

**Environments (please complete the following information):**
 - OS: [e.g. Linux]
 - go-zero version [e.g. 1.2.1]
 - goctl version github master

**More description**

**Comment by chenquan** [comment]
Use the goctl stable version: `go install github.com/zeromicro/go-zero/tools/goctl@latest`

**Comment by anqiansong** [comment]
在 #2368 已修复
> It has already fixed on #2368
